CHANGELOG.released.md

```diff
@@ -7,6 +7,25 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Calendar Versioning](http://calver.org/) `0Y.0M.MICRO`.
 For upgrade instructions, please check the [migration guide](MIGRATIONS.released.md).
 
+## [22.01.0](https://github.com/scalableminds/webknossos/releases/tag/22.01.0) - 2022-01-04
+[Commits](https://github.com/scalableminds/webknossos/compare/21.11.2...22.01.0)
+
+### Added
+- Added tagging support for datasets. [#5832](https://github.com/scalableminds/webknossos/pull/5832)
```

Comment:
This one could be a highlight maybe?

Reply:
Very good point! immediately forgot about the highlights-section after automating the changelog update process -.- I added a print-statement to the update script to remind about this.